Batch File/Batch File.sublime-syntax

```diff
@@ -27,8 +27,14 @@ contexts:
     - include: escaped_characters
     - include: variables
     - include: commands
-    - match: \b(?i)(?:goto|call|exit)\b
+    - match: \b(?i)exit\b
       scope: keyword.control.statement.dosbatch
+    - match: \b(?i)(goto|call)\s*(:)(?:(eof)|(\w+))
```

Comment:
This will remove highlighting for plain `goto` while typing. I suggest either adding a fallback or making the latter parts of the match optional.

Reply:
Good catch, thanks.